graphbolt/src/cnumpy.cc

```diff
@@ -0,0 +1,217 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file cnumpy.cc
+ * @brief Numpy File Fetecher class.
+ */
+
+#include "cnumpy.h"
+
+#include <torch/torch.h>
+
+#include <cstring>
+#include <regex>
+#include <stdexcept>
+
+namespace graphbolt {
+namespace storage {
+
+#define ALIGNMENT 4096
```

Comment:
```suggestion
constexpr int disk_alignment_size = 4096;
```
If you want, you could use all caps too. Depends on the code style.